src/codegen/codegen.test.ts

```diff
@@ -423,7 +425,7 @@ describe('Code generation', () => {
         params = { headers: {}, cookies: {} }
         url = http.url\`http://test.k6.io/api/v1/foo\`
         resp = http.request('POST', url, null, params)
-        check(resp,{'statusmatches200':(r)=>r.status===200,})
+        check(resp, { 'status equals recorded value': (r) => r.status === 200 })
```

Comment:
we discussed in the past that the check is more useful when it showcases what it's checking against, I think we should probably leave it like before `status equals 200`

Reply:
What if the recorded value has a different status (e.g. 201)?

Reply:
that part is generated so if the recorded value is `201` the check name would be as before: `status equals 201`

Reply:
basically: `status equals <value>: ... r.status === <value>`